pkg/clusterapi/controlplane.go

```diff
@@ -56,6 +56,7 @@ func (cp *ControlPlane[C, M]) UpdateImmutableObjectNames(
 ) error {
 	currentKCP := &controlplanev1.KubeadmControlPlane{}
 	err := client.Get(ctx, cp.KubeadmControlPlane.Name, cp.KubeadmControlPlane.Namespace, currentKCP)
+
```

Comment:
nit: new line?

Reply:
Ah, this was a line I probably touched while debugging but never discarded the changes properly. Will change it back.

Reply:
Fixed